forward_engineering/helpers/indexHelper.js

```diff
@@ -159,7 +159,7 @@ module.exports = app => {
 					return isParentActivated ? commentIfDeactivated(column, key) : column;
 				})
 				.join(',\n\t'),
-			indexName: index.keyIndex,
+			indexName: getIndexName({ name: index.name, schemaName: index.schemaName }),
```

Comment:
why you're taking `index.name` here instead of `index.keyIndex` for the `name` prop?